core/tests/unit/ldml/keyboards/k_006_backspace-test.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboardTest3 SYSTEM "../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboardTest3.dtd">
+<keyboardTest3 conformsTo="techpreview">
+  <info keyboard="k_006_backspace.xml" author="Team Keyboard" name="backspace" />
+  <tests name="backspace">
+    <test name="regex-test-0-a">
+      <startContext to="" />
+      <keystroke key="t" />
+      <keystroke key="a" />
+      <keystroke key="b" />
+      <backspace />
+      <check result="ta" /> <!-- TODO-LDML: #9760 Should be 't' due to backspace transform -->
```

Comment:
This test verifies that `<backspace />` works at all, #9760 to fix it for transforms